src/shared/buttercup/types.js

```diff
@@ -12,6 +12,7 @@ export const ImportTypes = {
   KEEPASS: 'keepass',
   LASTPASS: 'lastpass',
   ONE_PASSWORD: '1password',
+  BROWSERS: 'browsers',
```

Comment:
Probably best to use `CSV_GENERIC: 'csv_generic'` as this isn't just related to browsers :)